autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -68,19 +68,27 @@ def __init__(
         }
         self.prefix = f"[ExecutionManager|uid:{user_id}|gid:{graph_id}|nid:{node_id}]|geid:{graph_eid}|nid:{node_eid}|{block_name}]"
 
+    def _wrap(self, msg: str, **extra):
+        return f"{self.prefix} {msg} {extra}"
```

Comment:
Private methods below public ones please

Reply:
Updated, can you re-review?